src/app/components/PortraitVideoCarousel/PortraitVideoPromo/index.tsx

```diff
@@ -2,26 +2,64 @@
 import { jsx } from '@emotion/react';
 import { PortraitVideoPromoProps } from '#app/models/types/portraitVideo';
 import Image from '#app/components/Image';
-import Heading from '../../Heading';
+import Text from '#app/components/Text';
+import { Play } from '#app/components/icons';
+import VisuallyHiddenText from '#app/components/VisuallyHiddenText';
+import moment from 'moment';
+import formatDuration from '#app/lib/utilities/formatDuration';
+import { useContext } from 'react';
+import { ServiceContext } from '#app/contexts/ServiceContext';
 import styles from './index.styles';
 
 export default (item: PortraitVideoPromoProps) => {
-  const { images, headlines, onClick } = item;
+  const { images, headlines, video, onClick } = item;
+  const { translations } = useContext(ServiceContext);
 
   const imageUrl = images?.[0]?.url;
   const alt = images?.[0]?.altText || '';
   const headline = headlines?.promoHeadline || '';
+  const mediaISO8601Duration = video?.version.duration;
+  const mediaType = 'video';
+
+  const durationTranslation = translations?.media?.duration || 'Duration';
+  let durationString = '';
+  let durationSpokenString = '';
+  if (mediaISO8601Duration) {
+    const separator = ':';
+    const momentDuration = moment.duration(mediaISO8601Duration, 'seconds');
+    durationString = formatDuration({
+      duration: momentDuration,
+      padMinutes: true,
+    });
+    durationSpokenString = formatDuration({
+      duration: momentDuration,
+      separator,
+    });
+  }
+
+  const hiddenText = `${headline}, ${mediaType}, ${mediaISO8601Duration ? `${durationTranslation}  ${durationSpokenString}, ` : ''}Play ${mediaType}`;
 
   return (
     <button type="button" onClick={onClick} css={styles.button}>
+      <div css={styles.gradientOverlay}>
+        {mediaISO8601Duration && (
+          <div css={styles.durationContainer} aria-hidden="true">
+            <Play css={styles.playIcon} />
+            <time>
```

Comment:
Can we also set the `datetime` attribute on the `time` tag for consistency with other timestamps https://github.com/bbc/simorgh/blob/latest/src/app/legacy/psammead/psammead-timestamp/src/index.jsx#L37